### DISCUSSION TITLE
Poor search results

### DISCUSSION THREAD

**Discussion by thipokch** [community/community#27304] [Other Feature Feedback, Questions, & Ideas] [answered]
Search results / index are not up to date.
Partial words or different case doesn't show up.
Often show the least relevant relevant result first.

Often times I find myself googling for what I need.

**Comment by colinwm** [comment, marked as answer]
Are you using cs.github.com?

**Comment by thipokch** [comment]
@colinwm Thank's for the heads up. I just enrolled in the preview! Hopefully, that will quench my thirst for a better search results.
